### DISCUSSION TITLE
container tainted?

### DISCUSSION THREAD

**Discussion by Xel-Naha** [bpg/terraform-provider-proxmox#717] [Q&A] [answered]
hi there,

i've just started using this provider and looks promising, so thanks for that to begin with.

I've created a test container on my cluster and noticed that when I did a second apply the environment wanted to make some changes eventhough nothing had changed:

> Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
>   ~ update in-place
> 
> Terraform will perform the following actions:
> 
>   # module.envs.module.dev.proxmox_virtual_environment_container.ubuntu_container[0] will be updated in-place
>   ~ resource "proxmox_virtual_environment_container" "ubuntu_container" {
>       ~ description   = <<-EOT
>             test
>         EOT
>         id            = "101"
>         tags          = []
>         # (6 unchanged attributes hidden)
> 
>         # (3 unchanged blocks hidden)
>     }
> 
> Plan: 0 to add, 1 to change, 0 to destroy.

the tf config is pretty simple description = "test"

looking at the notes section it almost looks like a line feed is added:

![image](https://github.com/bpg/terraform-provider-proxmox/assets/100222761/3d3812bc-d9b1-49ed-a577-7b278b500cc5)

am I doing something wrong, or would this be a bug?

**Comment by bpg-dev** [comment]
Hi @Xel-Naha 👋🏼
This seems to be similar to https://github.com/bpg/terraform-provider-proxmox/pull/611 that was fixed in [v0.34.0](https://github.com/bpg/terraform-provider-proxmox/releases/tag/v0.34.0).
Which version of the provider are you running?

**Reply by Xel-Naha** [reply]
hi @bpg , this was the latest version as far as I can tell:

terraform {
  required_providers {
    proxmox = {
      source = "bpg/proxmox"
      version = "0.37.1"
    }
  }
}

**Reply by bpg-dev** [reply]
Thanks, that's a bug then. I'll create an issue.

**Reply by bpg-dev** [reply, marked as answer]
https://github.com/bpg/terraform-provider-proxmox/issues/721
